mars-sim-core/src/main/java/com/mars_sim/core/mission/MissionTravelStep.java

```diff
@@ -76,9 +76,7 @@ protected boolean execute(Worker worker) {
         boolean workedOn = false;
         if (vehicle.getOperator() == null) {
             Task operateVehicleTask = createOperateVehicleTask(vehicle, worker);
-            if (operateVehicleTask != null) {
```

Comment:
This check is needed because the Mission cannot always assign Tasks. So sometimes there is no Task associated with the mission that can be started.

Reply:
Perhaps it's okay here since createOperateVehicleTask() currently doesn't return null. 

It will either return a new instance of DriveGroundVehicle or throw a illegalArgumentException. 

I still don't like that we already start a DriveGroundVehicle task first.

Why can't we just test if it's assign-able first ? 

If it's not possible for this person to take on this task, we shouldn't even waste CPU cycles to create this task instance in the first place ?

Let's start a new ticket to address this.

The assignTask() method is being used in dozens of places. 

In the past, I've refactored the use of assignTask() for walking tasks.

Are there ways to make unit tests to test on various scenarios to see how much cpu utilization is wasted or saved by changing how it's being used ?